spec/features/idv/doc_auth/send_link_step_spec.rb

```diff
@@ -84,7 +84,7 @@
       throttle_type: :idv_send_link,
     )
 
-    Timecop.travel(Time.zone.now + idv_send_link_attempt_window_in_minutes.minutes) do
+    travel_to(Time.zone.now + idv_send_link_attempt_window_in_minutes.minutes + 1) do
```

Comment:
Specs like these where time travel is going to the beginning of a window would intermittently fail because it hadn't traveled far enough in time.

I'm not sure if it's due to partial seconds, a delay in Timecop, or something else, but the tests reliably pass by adding a second.

Reply:
🤷